routes/web.php

```diff
@@ -14,3 +14,9 @@
 Route::get('/', function () {
     return view('welcome');
 });
+
+Auth::routes();
+
+$test = 
```

Comment:
__[code_sniffer]__
* Whitespace found at end of line

<!-- Comment by [SideCI](https://sideci.com) -->